### ISSUE TITLE
[BUG] Source install includes .egg directories

### ISSUE THREAD

**Issue by itinneed2022** [graphite-project/graphite-web#2823]
**Describe the bug**
When trying to install from source, the path is extended to include ".egg" directories in /opt/graphite/webapp, /opt/graphite/lib

**To Reproduce**
I installed the requirements using the requirements.txt file:
(/usr/local/src/carbon -or- graphite)
python39 -m pip install -r requirements.txt
And then installed carbon:
python39 setup.py install

**Expected behavior**
/opt/graphite/webapp/ should contain content and graphite directories
This will let me run the django-admin migrate steps later in setup.

**Screenshots**
![image](https://github.com/graphite-project/graphite-web/assets/104800961/796b79ba-b4e9-41ad-862f-9f4e247188fd)

**Environment (please complete the following information):**
 - OS flavor: Red Hat 8.8
 - Graphite-web version 1.2.0
 - Django/Python version Python39, Django==4.2.7
 - Setup type Source

**Additional context**
I installed Django 4.2.7, using the requirements file. But I'm seeing Django 3.2.23 in the default directory:
/app/graphite/webapp/Django-3.2.23-py3.9.egg/

**Comment by itinneed2022** [comment]
Note, I just switched to Graphite 1.1.0, and the directory issue is gone.

**Comment by deniszh** [comment]
Yes, confirming that. For proper installation with recent python you need to call it like 
```bash
python39 setup.py install --single-version-externally-managed --root=/"
```
